extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -527,49 +563,30 @@ private function doUpdateDomains( &$request ) {
 	/**
 	 * doUpdateTags
 	 *
-	 * @access private
-	 * @author eloy@wikia-inc.com
-	 *
 	 * @param WebRequest $request  WebRequest instance
 	 * @param Boolean    $tag_id   set if removing, tag_id of removing tag
 	 *
 	 * @return mixed	info when change, null when not changed
 	 */
 	private function doUpdateTags( &$request, $tag_id = false ) {
-		#global $wgOut;
-
-		wfProfileIn( __METHOD__ );
-
-		$msg  = false;
 		$wfTags = new WikiFactoryTags( $this->mWiki->city_id );
 
-		if( $tag_id ) {
+		if ( $tag_id ) {
 			$tagName = $request->getText( "wpTagName" );
-			$newTags = $this->mTags->removeTagsById( array( $tag_id ) );
-			#$wgOut->addHTML('<pre>' . print_r($newTags,1) . "</pre>" );
 			$msg = "Tag {$tagName} removed";
-
-		}
-		else {
+		} else {
 			$stag = $request->getText( "wpTag", false );
 
-			if( $stag ) {
+			if ( $stag ) {
 				$before = $wfTags->getTags( true ); // from master
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)